app.js

```diff
@@ -46,24 +54,73 @@ var myColor = 'NO';
 
 if (color.toUpperCase() === myColor) {
   alert('Correct! I love green.');
+  count++;
 } else {
   alert('Nice try, but green is actually my favorite color.  I love being surrounded by plants!');
 }
 
-// var myMovement = ['yoga', 'hiking', 'qi gong', 'dancing'];
-// var movement = prompt('Name one of my favorite forms of exercise.');
-//
-// if (myMovement.indexOf(movement)!== -1) {
-//   alert('You\'re right, ' + movement + ' is one of my favorite forms of exercise.');
+var myMovement = ['yoga', 'hiking', 'qi gong', 'dancing'];
+var movement = prompt('Name one of my favorite forms of exercise.');
+
+if (myMovement.indexOf(movement)!== -1) {
+  alert('You\'re right, ' + movement + ' is one of my favorite forms of exercise.');
+  count++;
+} else {
```

Comment:
It's a style thing, but I think it's more readable and "better" to put the else on the next line, rather than next to the closing brace of the if.

It's a preference thing, so this isn't wrong, just something for you to consider.